engines/dgds/scene.cpp

```diff
@@ -750,11 +751,10 @@ bool Scene::runDragonOp(const SceneOp &op) {
 		engine->getScene()->addAndShowTiredDialog();
 		break;
 	case kSceneOpArcadeTick:
-		// TODO: Implement this properly! for now just
-		// set the global arcade state variable to the "skip" value.
-		warning("Setting arcade global to 8 (skip)");
-		g_system->displayMessageOnOSD(_("Skipping DGDS arcade sequence"));
-		engine->getGameGlobals()->setGlobal(0x21, 6);
+		// TODO: Add a configuration option to skip arcade sequence?
```

Comment:
The game itself allows you to do this via an in-game dialog, right? So we shouldn't need to do something extra here

Reply:
Yeah, it offers to skip after you fail 5 times in a row.  I guess it's not that hard to fall into a hole repeatedly if you want to skip it.